Octokit.Tests/Clients/TeamDiscussionsClientTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using NSubstitute;
+using Xunit;
+
+namespace Octokit.Tests.Clients
+{
+    /// <summary>
```

Comment:
I think this is an old comment from where this was copied from, and it doesnt really make too much sense (referring to ApiConnectionTests.cs).  I think we can just remove this whole <summary> section